Cargo.toml

```diff
@@ -8,7 +8,7 @@ cached = "0.46.1"
 clap = "4.4.8"
 clap_complete = "4.4.4"
 convert_case = "0.6.0"
-extism = { version = "0.5.4" }
+extism = { git = "https://github.com/extism/extism.git", branch = "backport-cache"}
```

Comment:
I can't publish to crates.io when using the git pattern.

Reply:
```suggestion
extism = "0.5.5"
```

Reply:
We released a new stable you can try